staging/src/k8s.io/apiserver/pkg/admission/plugins.go

```diff
@@ -127,6 +127,8 @@ func splitStream(config io.Reader) (io.Reader, io.Reader, error) {
 // the given plugins.
 func (ps *Plugins) NewFromPlugins(pluginNames []string, configProvider ConfigProvider, pluginInitializer PluginInitializer, decorator Decorator) (Interface, error) {
 	handlers := []Interface{}
+	mutationPlugins := []string{}
```

Comment:
can we just have a counter for both, since we don't really need the contents of the array?

Reply:
I think we need the contents of the array.
We need to log the enabled admission controller name.

Reply:
ack thanks